GETTING_STARTED.md

```diff
@@ -1,109 +1,86 @@
-    {'firstname': 'Han', 'lastname': 'Solo', 'email': 'han.solo@macrometa.io'},
-    {'firstname': 'Bruce', 'lastname': 'Wayne', 'email': 'bruce.wayne@macrometa.io'}
-]
+## Update Document
+The [update](https://macrometa.com/docs/queryworkers/c8ql/got-tutorial/c8ql-crud#update-documents) operation can partially update a document in a collection. The document must contain the attributes and values to be updated and the _key attribute to identify the document to be updated. For this example the email for *Han* will be updated.
 
-client.insert_document(collection_name=collname, document=docs)
+```python
+client.update_document(collection_name='employees', document={'_key': 'Han', 'email': 'han@updated_macrometa.io'})
+```
 
-# insert documents from file
-client.insert_document_from_file(collection_name=collname,
-                                 csv_filepath="/home/user/test.csv")
+## Delete Document
+The [delete](https://macrometa.com/docs/queryworkers/c8ql/got-tutorial/c8ql-crud#delete-documents) operation is used to remove a document from a collection. For this example *Han* will be removed from the **employees** collection.
+```python
+client.delete_document(collection_name='employees', document={'_key': 'Han'})
+```
 
-# Add a hash Index
-client.add_hash_index(collname, fields=['email'], unique=True)
+## Working with C8QL
+Working with data can be complex. CRUD operations usually need more logic or conditions to reach the desired results. At Macrometa the [C8 query language (C8QL)](https://macrometa.com/docs/queryworkers/c8ql/) can be used to create, retrieve, modify and delete data that are stored in the Macrometa geo-distributed fast data platform.
+**Check out the [operators](https://macrometa.com/docs/queryworkers/c8ql/) and [examples](https://macrometa.com/docs/queryworkers/c8ql/examples/) in Macrometa**
 
-# --------------------------------------------------------------
+Let's [FILTER](https://macrometa.com/docs/queryworkers/c8ql/operations/filter) results by **role** and **age** and return a custom object using [ C8QL](https://macrometa.com/docs/queryworkers/c8ql/):
+```python
+query = "FOR doc IN employees FILTER doc.role == 'Manager' FILTER doc.age > 30 RETURN {'Name':doc.firstname,'Last Name':doc.lastname,'Email':doc.email}"
+cursor = client.execute_query(query=query)
+docs = [document for document in cursor]
+```
 
-# Query a collection
-print("query employees collection...")
-query = 'FOR employee IN employees RETURN employee'
-cursor = client.execute_query(query)
+It might not always be necessary to return all documents that a FOR loop would normally return. In those cases, you can limit the number of documents with a [LIMIT()](https://macrometa.com/docs/queryworkers/c8ql/operations/limit) operation. For this case, it will be used along with the [SORT()](https://macrometa.com/docs/queryworkers/c8ql/operations/sort) operation.
+```python
+query = "FOR doc IN employees SORT doc.age LIMIT 2 RETURN doc"
```

Comment:
I think that while explaining limit offset also plays a part since they are frequently used together so maybe you can add a small one-liner LIMIT offset, count example as well here